dev-libs/pigpio/pigpio-67.ebuild

```diff
@@ -1,7 +1,7 @@
 # Copyright 1999-2018 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
```

Comment:
Don't EAPI bump old ebuild unnecessarily. This is a potentially breaking change.

Reply:
That was specifically referring to -67, I'm sorry if it was confusing. EAPI 7 for -68 is desirable.